hdfs-commons/src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -56,8 +57,12 @@ private HDFSConstants() {
   public static final String ZK_FRAMEWORK_ID_KEY = "FrameworkId";
   public static final Integer ZK_MUTEX_ACQUIRE_TIMEOUT_SEC = 30;
   public static final Integer CURATOR_MAX_RETRIES = 3;
+
   public static final String NN_STATUS_KEY = "status";
   public static final String NN_STATUS_INIT_VAL = "initialized";
   public static final String NN_STATUS_UNINIT_VAL = "uninitialized";
```

Comment:
They are still used in mechanism, for passing task status to the scheduler:
org.apache.mesos.hdfs.util.TaskStatusFactory#getNameNodeInitStatus
org.apache.mesos.hdfs.state.HdfsState#nameNodesInitialized
So can't remove them.